.github/workflows/publish-homebrew.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to Homebrew
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
```

Comment:
Since the download url is from npm registry, when a release is created, the `mislav/bump-homebrew-formula-action` action might be executed **before** the other workflow finishes uploading to npm. Thus, this workflow will fail.

In this case, instead of release trigger, I think it's better to use [workflow_run trigger](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run).

Reply:
updated